### PR TITLE
feat(useful-tools/networks): Update as requested

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: src/docs/useful-tools/networks.md [hunk 1]
```diff
@@ -79,25 +87,14 @@ Links to download data directories will be posted here as soon as they are avail
...
 
-- Bedrock Data Directory: Will be posted here as soon as it is available
-- Legacy Geth Data Directory: Will be posted here as soon as it is available
-- [Configuration parameters](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/deploy-config/mainnet.json)
-- [Docker images (`op-node` and `op-geth`)](https://console.cloud.google.com/artifacts/docker/oplabs-tools-artifacts/us/images)
-- [Legacy geth](https://hub.docker.com/layers/ethereumoptimism/l2geth/0.5.29/images/sha256-e40ea183c[…]e981408ccf9c07191667554c48b620e5eabc61521c3b3?context=explore)
+- Bedrock Data Directory: Will be posted here as soon as it is available.
+- Legacy Geth Data Directory: Will be posted here as soon as it is available.
+- [Configuration parameters](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/deploy-config/mainnet.json).
+- Docker image for `op-geth`: Will be posted here as soon as it is available.
```
Comment (K-Ho) [inline src/docs/useful-tools/networks.md:93]: op-geth release is already live for launch: https://github.com/ethereum-optimism/op-geth/releases/tag/v1.101105.2

File: src/docs/useful-tools/networks.md [hunk 2]
```diff
@@ -182,6 +157,29 @@ To see the full list of providers visit [Node & API Providers](./providers.md).
...
+| Withdrawal period | 2 seconds |
+
+
+
+#### Links
+
+- [Bedrock Data Directory](https://storage.googleapis.com/oplabs-goerli-data/goerli-bedrock.tar)
+- [Legacy Geth Data Directory](https://storage.googleapis.com/oplabs-goerli-data/goerli-legacy-archival.tar)
+- [Configuration parameters](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/deploy-config/goerli.json)
+- [Docker image for `op-geth`](https://console.cloud.google.com/artifacts/docker/oplabs-tools-artifacts/us/images/op-geth).
```
Comment (K-Ho) [inline src/docs/useful-tools/networks.md:178]: instead link to https://github.com/ethereum-optimism/op-geth/releases

Comment (K-Ho) [reply src/docs/useful-tools/networks.md:178]: Also rename to something like  "Latest releases for `op-geth`"

File: src/docs/useful-tools/networks.md [hunk 3]
```diff
@@ -182,6 +157,29 @@ To see the full list of providers visit [Node & API Providers](./providers.md).
...
+- [Docker image for `op-node`](https://console.cloud.google.com/artifacts/docker/oplabs-tools-artifacts/us/images/op-node)
```
Comment (K-Ho) [inline src/docs/useful-tools/networks.md:179]: Instead link to https://github.com/ethereum-optimism/optimism/releases

Comment (K-Ho) [reply src/docs/useful-tools/networks.md:179]: "Latest releases for `op-node`"

**Review by K-Ho — APPROVED** [review]
Approving in advance assuming we make those small changes.

File: src/docs/useful-tools/networks.md (hunk 2 above)
Comment (qbzzt) [reply src/docs/useful-tools/networks.md:178]: Is there a reason not to direct them to the same `op-geth` as mainnet?

**Review by OPMattie — APPROVED** [review]

**Pull request merged**